_data/aakashbaranwal/aakashbaranwal.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aakash Baranwal
+institution: Dayalbagh Educational Institute
+profile_pic: aakashbaranwal.jpg
```

Comment:
I just noticed this! Image file path name seems incorrect, please change it from `aakashbaranwal.jpg` to `aakashbaranwal.png`

Reply:
Okay, I'll just correct it, please give me some time